File: server/api/pkg/domain/instance_info.go
package domain

type InstanceInfo struct {
	Id                    string  `json:"id"`
	Host                  string  `json:"host"`
	Name                  *string `json:"name"`
	Description           *string `json:"description"`
	ClientMaxBodyByteSize *int64  `json:"clientMaxBodyByteSize"`
	IconUrl               *string `json:"iconUrl"`
	ThemeColor            *string `json:"themeColor"`
}

File: modules/model/src/main/java/net/pantasystem/milktea/model/instance/InstanceInfo.kt
package net.pantasystem.milktea.model.instance

data class InstanceInfo(
    val id: String,
    val host: String,
    val name: String?,
    val description: String?,
    val clientMaxBodyByteSize: Long?,
    val iconUrl: String?,
    val themeColor: String?,
)

